src/fundus/publishers/de/rbb24.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class RBB24Parser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = XPath("//div[contains(concat(' ', @class , ' '), ' textblock ')]")
```

Comment:
This way, the text gets extracted as a full block. If you change it to `//div[contains(concat(' ', @class , ' '), ' textblock ')]/p` the amoung of extracted text stays the same, but it's separated into paragraphs. Btw, this might be a simpler solution, which has the same result: `//div[@class='textblock']/p`